core/src/main/java/org/kiwix/kiwixmobile/core/utils/BookUtils.kt

```diff
@@ -15,46 +15,44 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
-package org.kiwix.kiwixmobile.core.utils;
+package org.kiwix.kiwixmobile.core.utils
 
-import java.util.HashMap;
-import java.util.Locale;
-import java.util.Map;
+import java.util.HashMap
+import java.util.Locale
 
 /**
  * Created by mhutti1 on 19/04/17.
  */
-
-public class BookUtils {
-
-  public final Map<String, Locale> localeMap;
+class BookUtils {
+  val localeMap: Map<String, Locale>
 
   // Create a map of ISO 369-2 language codes
-  public BookUtils() {
-    String[] languages = Locale.getISOLanguages();
-    localeMap = new HashMap<>(languages.length);
-    for (String language : languages) {
-      Locale locale = new Locale(language);
-      localeMap.put(locale.getISO3Language(), locale);
-    }
+  init {
+    val languages = Locale.getISOLanguages()
```

Comment:
no need for the init

```
val localeMap =Locale.getISOLanguages().associateBy {...}
```
I think works off the top of my head but there is definitely a function to make this more succint and fucntional